packages/edit-post/src/store/private-selectors.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+import { createRegistrySelector } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import { store as editorStore } from '@wordpress/editor';
+
+export const getEditedPostTemplateId = createRegistrySelector(
```

Comment:
This is the same logic that was in `getEditedPostTemplate` but I moved it to this new selector because we don't really need the full object.